python/tank/bootstrap/manager.py

```diff
@@ -546,15 +560,17 @@ def get_pipeline_configurations(self, project):
             If ``None``, this will enumerate the pipeline configurations
             for the site configuration.
         :type project: Dictionary with keys ``type`` and ``id``.
+        :param list pc_entities: A list of PipelineConfiguration entity dictionaries. This
```

Comment:
Just a subjective opinion, but maybe this could be named something like `external_data` instead? To highlight that instead if having the method fetch the data, it just operates directly on this external data that is passed? Feeel free to ignore this if you disagree! :)